src/SharedFrameworkValidation/SharedFrameworkValidation.proj

```diff
@@ -82,10 +98,23 @@
       <_CloneRepoLines Include="popd" />
     </ItemGroup>
 
+    <!-- Unix script -->
+    <ItemGroup Condition="'$(TargetsUnix)'=='true'">
+      <_CloneRepoLines Include="#!/usr/bin/env bash" />
```

Comment:
BTW, if you're building up a multiline thing you can put multiple lines in like
```
<prop>
foo
bar
</prop>

or
<item Include="
foo
bar
</item>
```

but either is fine of course...

Unfortunately of course you can't use CDATA to avoid the %22 type escaping, because you're using items so it's an attribute value.